src/kinsol/kinsol.c

```diff
@@ -2375,17 +2382,7 @@ void KINInfoHandler(const char *module, const char *function,
                     char *msg, void *data)
 {
   KINMem kin_mem;
-
-  /* data points to kin_mem here */
-
   kin_mem = (KINMem) data;
-#ifndef NO_FPRINTF_OUTPUT
-  if (kin_mem->kin_infofp != NULL &&
-      kin_mem->kin_infofp != kin_mem->kin_sunctx->logger->info_fp) {
-    fprintf(kin_mem->kin_infofp,"\n[%s] %s\n",module, function);
-    fprintf(kin_mem->kin_infofp,"   %s\n",msg);
-  }
-#endif
 #if SUNDIALS_LOGGING_LEVEL >= SUNDIALS_LOGGING_INFO
   SUNLogger_QueueMsg(KIN_LOGGER, SUN_LOGLEVEL_INFO,
     "KINSOL::KINInfoHandler", function, "%s", msg);
```

Comment:
Looks like the `module` input is lost, should this should this be something like `[KINSOL::<module>]`

Reply:
Also this could probably be moved into `KINPrintInfo` and `kin_ihfun` removed since the user can't override it any more

Reply:
The `module` input is not really relevant anymore though, is it? Its also not consistent with the rest logging in the other packages.

Reply:
`module` in this case is really the function that logged the message i.e., `KINFullNewton` so we'd replace `"KINSOL::KINInfoHandler"` with `"KINSOL::KINFullNewton"` which would align with the other packages e.g. `"ARKODE::arkEvolve"`

Reply:
Looks like I'm confusing the inputs. You're right `module` can be ignored. It should be `"KINSOL::<function>"` to align with the other messages and I guess where `function` is now would be empty.

Reply:
Yes, so we just need to drop `::KINInfoHandler`.  The function will get added since the function name gets passed to QueueMsg which appends it.